### DISCUSSION TITLE
RxNorm to complete NDC HCPCS walk

### DISCUSSION THREAD

**Discussion by Bridg109** [coderxio/sagerx#245] [Use cases]
# Proposal
CMS quarterly HCPCS does not contain all NDC available for a specific HCPCS code. Use RxNorm mapping to complete CMS mapping

# Rationale
assessment of NDC level claims data will has potential for missing values if CMS quarterly file alone is used. Completing the mapping using similarity test in RxNorm will improve analysis from this data set

**Comment by jrlegrand** [comment]
This sounds like a dbt intermediate model depending on RxNorm and CMS quarterly HCPCS updates (https://coderx.io/sagerx/source-data/cms-alpha-numeric-hcpcs) which we don't currently have in SageRx. I think we need a customer for this data before we go after it.
